internal/ingress/metric/collectors/socket.go

```diff
@@ -217,6 +217,7 @@ func NewSocketCollector(pod, namespace, class string, metricsPerHost bool, bucke
 	}
 
 	sc.metricMapping = map[string]interface{}{
+		prometheus.BuildFQName(PrometheusNamespace, "", "requests"):                 sc.requests,
```

Comment:
additionally during unit-testing, i also discovered that `requests` was missing from here (this was the only one metric that was missing).